### ISSUE TITLE
Collection is happening twice if i click the submit button in the collection sheet twice.

### ISSUE THREAD

**Issue by MathiDuruva** [mifos-x-customization/temp-os#12]
steps:
1. go to collection sheet and enter the details.
2. On click of submit the amount should be collected for that installment.
3. But when i click two times the same button continuously it is getting collected for the next installment also.
 expected behavior:
the submit button should be allowed to click only once for one center

**Comment by maektwain** [comment]
Ok, so I have a centre LBS in my demo instance, it has a meeting date for 07 October 2019, now in that case which is day before 7 , I am trying to submit the repayment it does not allow me to do that, it says no future date is allowed. 

1. I think you have to understand this is not an issue, you cannot and should not submit since, today date if you try to submit the repayment for today, it will lookup in the db for present date and then If it does not match then it will pay off for next due date.

Could you give me an example on the server.

**Comment by Meena88** [comment]
no, you didn't get the point. lets say there is an center "TEST" with its 3rd installment as today , that is 15th oct 2019 , im selecting that particular center in the collection sheet and then clicking on the submit button twice (i.e. a double click on the submit button). This double click is landed as the collection for 3rd installment as well as for 4th installment in repayment schedule for the same date , 15th oct 2019 ,, (which need to actually happen on 22nd oct 2019 . 
same way when i click triple  times the repayment will be added to 3rd, 4th , 5th installments for all the clients in that particular center

**Comment by Meena88** [comment]
this is a major issue, which should not happen ever, so what i want you to do here is, make a condition in the code, that the submit button in productive collection sheet should allow only one click as input from user,
even the user makes double click or triple clicks only the first click must be considered for the input as only one colleciton ,must be made in the repayment for the selected date

**Comment by maektwain** [comment]
This is not a valid argument , you need not to use double click. I am sorry this is not a feature nor a bug . its simply a human perception of using a single click or double, I would like to know why do you want to use double click at all, when you know single click can do the job for you.

**Comment by Meena88** [comment]
i dont think every requirement should sound 100% convincing to a developer, You cannon control everything when human intervention occurs. Even though we inform our users not to make the double click, on the course of their work due to the fast movement, they are making this mistake often, so as a developer, im asking you to make a condition ,so that there will be no such case occurrence.  i have explained clearly my point. if you still dont feel this is reasonable from your end, no problem, just consider this as an important requirement from your client side and complete it for us.

**Comment by maektwain** [comment]
Makes sense, I have closed this and you can test it once deployed.

Thanks
